.github/workflows/ci_test-full.yml

```
@@ -127,6 +120,7 @@ jobs:
           pip install --no-cache-dir -r requirements/horovod.txt
         fi
         horovodrun --check-build
+        python -c "import horovod.torch; horovod.torch.nccl_built()"
```

`horovod.torch.nccl_built()` returns an integer so without an assertion this does nothing

And anyways we don't want to check this because this workflow is CPU

```suggestion
        python -c "import horovod.torch"
```